apiserver/routers.go

```diff
@@ -41,5 +42,34 @@ func (api *APIServer) NewRouter(ctx context.Context) *mux.Router {
 		Name("GetDistroSigImageConfig").
 		HandlerFunc(api.GetDistroSigImageConfig)
 
+	router.Methods("GET").Path("/healthz").Name("healthz").HandlerFunc(healthz)
+
+	// global timeout and panic handlers.
+	router.Use(timeoutHandler(), recoveryHandler())
+
 	return router
 }
+
+func healthz(w http.ResponseWriter, r *http.Request) {
+	handleOK(w, r)
+}
+
+func setHeaders(w http.ResponseWriter) {
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	w.Header().Set("X-Content-Type-Options", "nosniff")
```

Comment:
no idea why this is needed.

Reply:
it's not needed, I like to be a good citizen and set content type headers correctly. it's copy paste from myself years ago :P

will remove for the sake of keeping this small.

Reply:
or will shove in handleOK and drop the nosniff since that's really for browsers.